src/main/java/net/snowflake/client/core/bind/BindUploader.java

```diff
@@ -386,6 +384,15 @@ private void uploadStreamInternal(
     return sb.toString().getBytes(UTF_8);
   }
 
+  private String getCreateStageStatement() {
+    return "CREATE TEMPORARY STAGE IF NOT EXISTS "
+        + session.getSfConnectionHandler().getBindStageName()
+        + " file_format=("
+        + " type=csv"
+        + " field_optionally_enclosed_by='\"'"
+        + ")";
+  }
```

Comment:
I assume this function is going to be overwrited by sp jdbc's implementation right?

Reply:
Actually, no. I changed this to a `final String` field that is initialized in the constructor. All it needs is the bind-stage name.